dependencies.yaml

```diff
@@ -0,0 +1,198 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: conda
+    matrix:
+      cuda: ["11.5", "11.4", "11.2", "11.0"]
+      arch: [x86_64]
+    includes:
+      - build
+      - cudatoolkit
+      - develop
+      - doc
+      - notebook
+      - py_version
+      - run
+      - test_python
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - dask/label/dev
+  - conda-forge
+  - nvidia
+dependencies:
+  build:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - cmake>=3.23.1,!=3.25.0 # build dependency not listed in conda dev environment
+          - cuda-python>=11.7.1,<12.0
+          - scikit-build>=0.13.1
+          - cython>=0.29,<0.30 # build dependency not listed in conda dev environment
```

Comment:
Reminder for us to remove these comments once we decide whether to keep/remove these dependencies